docs/logs/correlation/Program.cs

```diff
@@ -0,0 +1,49 @@
+﻿// <copyright file="Program.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System.Diagnostics;
+using Microsoft.Extensions.Logging;
+using OpenTelemetry;
+
+public class Program
+{
+    private static readonly ActivitySource DemoSource = new ActivitySource("OTel.Demo");
+
+    public static void Main()
+    {
+        using var tracerProvider = Sdk.CreateTracerProviderBuilder()
+            .AddSource("OTel.Demo")
+            .Build();
+
+        using var loggerFactory = LoggerFactory.Create(builder =>
+        {
+            builder
+            .AddJsonConsole(options => { options.IncludeScopes = true; })
+            .Configure(options => options.ActivityTrackingOptions =
```

Comment:
@reyang It looks like the default is TraceId | SpanId | ParentId should we have a note about that?

https://github.com/dotnet/runtime/blob/6072e4d3a7a2a1493f514cdf4be75a3d56580e84/src/libraries/Microsoft.Extensions.Hosting/src/Host.cs#L120-L122

Reply:
Let me put some description in the README.md.

Reply:
Updated the `README.md` file. Later when `ActivityTrackingOptions` is publicly documented we will link to the official doc.
For now I just listed all the values (instead of describing what is "default").